test.properties

```diff
@@ -4112,11 +4112,13 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
     playwright.test.project[playwright-js-tomcat90-mysql57-jdk8][content-management]=\
         announcements-web,\
         blogs-web,\
+        content-dashboard-web,\
         document-library-web,\
         journal-web,\
         knowledge-base-web,\
         message-boards-web,\
-        notification-web
+        questions-web,\
```

Comment:
question:

the deletion of  notification-web module was made on purpose?

Reply:
Yep because I only found tests of Business Process Management Product Team

FF - https://liferay.atlassian.net/browse/LPD-11165
Test - https://liferay.atlassian.net/browse/LPD-23257

Reply:
Our notification module is under notification`s`-web I´m not sure if it is the same module 🤔

Reply:
I added a commit c0bce2e546a158e78e5f4e948ecec219a7dc00cd using the right name.

It's added wrongly in #5512.